tests/system/providers/google/cloud/automl/example_automl_translation.py

```diff
@@ -24,7 +24,7 @@
 from datetime import datetime
 from typing import cast
 
-from google.cloud import storage
+from google.cloud import storage  # type: ignore[attr-defined]
```

Comment:
Hmm is the Google API module not annotated well or something? Maybe we should ignore the entire thing in Mypy config instead.

Reply:
I tried to mov it to setup.cfg. it seems that storage is simply not yet mypy-ready:
https://github.com/googleapis/python-storage/issues/393

However, I have a hard time finding an easy way to exclude it in config more selectively. 
I tried few ways and I seem I cannot do it other than (or I do not know a better way):

I tried few approaches (ignore_missing_imports on google.cloud level, ignore_errors on google.cloud.storage level etc. etc. .. Seems that ignoring this one attr-defined is better as it is very selective (and also it is close to the source, which is kinda better than somewhat distant specification in config).

I added a comment to https://github.com/googleapis/python-storage/issues/393 as an explanation and I think it should be "good-enough".